kiwi/xml_state.py

```diff
@@ -470,6 +470,18 @@ def get_build_type_vmdvd_section(self):
             if vmdvd_sections:
                 return vmdvd_sections[0]
 
+    def get_build_type_vmconfig_entries(self):
```

Comment:
@schaefi shouldn't it return an empty list in case there is no machine section defined? Then we are sure it is iterable.